Configuration/StandardSequences/python/ReconstructionCosmics_cff.py

```diff
@@ -69,6 +69,7 @@
                                              jetsCosmics*
                                              muonsCosmics*
                                              regionalCosmicTracksSeq*
+                                             cosmicCDCTracksSeq*
```

Comment:
So, the fact is that here we are using a different approach, not the regional one. So I would keep this name.